### DISCUSSION TITLE
How to run an equilibrium SSP585 ECHAM6-wiso standalone simulation on ollie?

### DISCUSSION THREAD

**Discussion by QinggangGao** [esm-tools/esm_tools#744] [Models]
Dear developers

I plan to run an equilibrium SSP585 ECHAM6-wiso standalone simulation on ollie. The first step is to run a standard ssp585 simulation without changing any default boundary conditions. However, I met a problem with it, could you please advise on it? Below are detailed descriptions.

ESM_TOOLS version: 6.11.4
ECHAM version: 6.3.05p2-wiso
yaml scripts: `/work/ollie/qigao001/a_basic_analysis/d_model_simulations/2_echam6/2.2_ssp585/ssp5_echam6_1d.yaml`
Error message while issuing esm_runscripts. command:

> qigao001> esm_runscripts /work/ollie/qigao001/a_basic_analysis/d_model_simulations/2_echam6/2.2_ssp585/ssp5_echam6_1d.yaml -e test
> Copying standard yamls from:  /home/ollie/qigao001/esm_tools/configs/
> Copying standard namelists from:  /home/ollie/qigao001/esm_tools/namelists/
> /work/ollie/qigao001/a_basic_analysis/d_model_simulations/2_echam6/2.2_ssp585/ssp5_echam6_1d.yaml
> First year, checking if we need to compile...
> 
> STARTING SIMULATION JOB!
> Experiment ID = test
> Setup = echam
> Experiment is installed in:
>        /work/ollie/qigao001/output/echam-6.3.05p2-wiso/ssp5//test
> 
> 
> ERROR: Missing Configuration
> The input file variable LU_HIST of forcing_sources can not be fully resolved:
> 
> /work/ollie/pool/JSBACH/input/r0010/T63/New_Hampshire_LCC/hist_harvest/LUH_harvest_T63_1850.nc:
>     to: 1850
> /work/ollie/pool/JSBACH/input/r0010/T63/New_Hampshire_LCC/hist_harvest/LUH_harvest_T63_@YEAR@.nc:
>     from: 1851

There is a similar problem in another ticket: https://github.com/esm-tools/esm_tools/discussions/673. But I cannot find an answer there.

That's for a different model but with the same error message. The suggested link there does not work anymore.

Regards, Qinggang

**Comment by QinggangGao** [comment]
Hi, I also set up the environment on Levante. If it's easier to run ssp585 simulation on Levante, can you please let me know how to configure it on Levante?

**Comment by mandresm** [comment]
Turned into #746 issue, and replied there
